README.md

```diff
@@ -304,6 +311,79 @@ total-size: false
 symlink-arrow: ⇒
 ```
 
+## Theme
+
+`lsd` can be configured with a theme file to set the colors.
+
+Theme can be configured in the [configuration file](#configuration)(color.theme),
+The valid theme configurations are:
+- `default`: the default color scheme shipped in `lsd`
+- theme-file-name(yaml): use the theme file to specify colors
+
+when configured with the `theme-file-name` which is a `yaml` file,
+`lsd` will look up the theme file in the following way:
+- relative name: check the XDG Base Directory
+- absolute name: use the file path and name to find theme file
```

Comment:
Not sure if absolute will be of all that much use, but I guess it couldn't hurt.

Reply:
the absolute path will be useful in tests scenario, and some others like trying newly downloaded themes, just like you said, it couldn't hurt